backend/src/main/java/com/bakdata/conquery/models/jobs/Job.java

```diff
@@ -30,6 +30,9 @@ public boolean isCancelled() {
 
 	public abstract void execute() throws Exception;
 	public abstract String getLabel();
+	public void destroy(){
```

Comment:
Wenn du die Methode behalten musst, mach bitte Doku dran, wofür sie da ist und in welchen Fällen sie gerufen wird.

Reply:
Die Methode habe nun rausgenommen